Yoast/wordpress-seo
# Consistently apply Help Tab styling

Introduces a CSS class to use in title styling in Help Center tabs.

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

* Introduce a new CSS class based on the existing class that matches the style we want to use.

## Test instructions

This PR can be tested by following these steps:

* Open the Help Center on the `Search Appearance` / `Post edit` / `Term edit` pages
* Go to the "Snippet variables" tab
* See the titles using the same styling in all tabs

## Quality assurance
